egs/chime6/asr1/conf/preprocess.yaml

```diff
@@ -0,0 +1,21 @@
+process:
```

Comment:
this looks similar to `specaug.yaml`.
could you use either of them?

Reply:
We made it specaug.yaml